imports/marker/client.lua

```diff
@@ -0,0 +1,99 @@
+lib.marker = {}
+
+local defaultRotation = { x = 0, y = 0, z = z }
+local defaultDirection = { x = 0, y = 0, z = z }
+local defaultColor = { r = 255, g = 255, b = 255, a = 100 }
+local defaultSize = { width = 2, height = 1 }
+
+---@enum MarkerType
+MarkerTypes = {

```

Comment:
Yes so people can use MarkerType.Chevron, how else could we do this?

Reply:
I can think of two possible options, linden should give the final say

but either something like lib.marker.chevron
or make an alias for autocompletion (unsure if one can be inferred from your enum instead of typing one by one)

```lua
---@alias MarkerType
---| '"UpsideDownCone"'
---| '"VerticalCylinder"'
---| '"ThickChevronUp"'
---| '"ThinChevronUp"'
```

![image](https://github.com/overextended/ox_lib/assets/21954193/1e68b1c9-b9e2-43ce-a92f-f8f5ba172e60)